apps/src/p5lab/AnimationPicker/AnimationUploadButton.jsx

```diff
@@ -26,43 +23,25 @@ export function UnconnectedAnimationUploadButton({
   onUploadClick,
   shouldWarnOnAnimationUpload,
   isBackgroundsTab,
-  teacherHasConfirmedUploadWarning,
   inRestrictedShareMode,
   refreshInRestrictedShareMode,
-  refreshTeacherHasConfirmedUploadWarning,
   showingUploadWarning,
   exitedUploadWarning,
-  currentUserType,
 }) {
   const [isUploadModalOpen, setIsUploadModalOpen] = useState(false);
   const [isPublishedWarningModalOpen, setIsPublishedWarningModalOpen] =
     useState(false);
 
-  // Some of the behavior (particularly in the confirmation dialog) is conditional
-  // on whether a student or teacher is uploading.
-  // Teachers see a warning not to upload PII, and once they confirm this warning
-  // we save that state to their project and don't show the warning again.
-  // Students see a warning to not upload PII as well as a statement that they will not be able
+  // Users see a warning to not upload PII as well as a statement that they will not be able
   // to share their project if they upload -- we also save this state to their project and don't show the warning again.
-  const isTeacher = currentUserType === 'teacher';
-  let hasConfirmedWarning, updateWarningState;
-  if (isTeacher) {
-    hasConfirmedWarning = teacherHasConfirmedUploadWarning;
-    updateWarningState = () => {
-      project.setTeacherHasConfirmedUploadWarning(true);
-      refreshTeacherHasConfirmedUploadWarning();
-    };
-  } else {
-    hasConfirmedWarning = inRestrictedShareMode;
-    updateWarningState = () => {
-      project.setInRestrictedShareMode(true);
-      // redux setting, for use in share/remix
-      refreshInRestrictedShareMode();
-    };
-  }
+  const updateWarningState = () => {
+    project.setInRestrictedShareMode(true);
+    // redux setting, for use in share/remix
```

Comment:
Not sure what this means; worth elaborating on it a bit?

Reply:
Updated 👍